components/fcc_gne_reference/fcc_gne_reference.module

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Module containing GNE Reference component.
+ */
+
+/**
+ * Implements hook_ctools_plugin_directory().
+ */
+function fcc_gne_reference_ctools_plugin_directory($module, $plugin) {
+  if ($module == 'ctools' && !empty($plugin)) {
+    return "plugins/$plugin";
```

Comment:
if you have one plugin let's say `content_type`, it's good to check against that only. thus we can avoid unnecessary directory scans.

in case of multiple plugin types use `in_array()`